### ISSUE TITLE
Snapping for rotation

### ISSUE THREAD

**Issue by wowbagger84** [Unity-Technologies/2d-spriteshape-samples#8]
I don't know if this is the right place for this request, but I can't get into the forums right now so I'll just post here for now. Please let me know where to post this if I'm in the wrong place.

When moving a point on a sprite shape spline you have the option to snap the points to the grid. I would love to have a checkbox for snapping tangent-handles to the grid as well. It would make it so much easier to create rounded corners that always becomes the same.

Having your points on the grid and using straight lines works somewhat but are not always ideal.

**Comment by worldsayshi** [comment]
Yes, I came here to post this as well. Although the title threw me off a it. To be clear for someone skimming this, these things:

![tangents](https://user-images.githubusercontent.com/930909/80040804-258ee180-84fb-11ea-9e50-3d9b91c5017b.png)

Anyway, so I looked into the code. Looks like it needs to be fixed in `ShapeEditor.HandleDragTangent` where we need a call to Snap in a similar way as for `ShapeEditor.HandleDragPoints`.

**Comment by worldsayshi** [comment]
Ops, I think I have a really old version. Strange that, Help->Check for updates told me I was up to date. Guess I need to update from the home page.

Edit: I updated to Unity 2019.3.10f1 and 2D Spriteshape 3.0.10. Still no tangent snapping feature though. The above code references are no longer relevant.
